android/java/res/xml/brave_news_preferences.xml

```diff
@@ -19,11 +19,16 @@
         android:summaryOn="@string/brave_news_optin_subtitle"
         android:summaryOff="@string/brave_news_optin_subtitle" />
 
-<!--     <PreferenceCategory
+    <PreferenceCategory
```

Comment:
what was that commented out element :+1:

Reply:
@AlexeyBarabash i think it was an old formatting/commit thing, nothing important should be missing